pkg/kubelet/apis/cri/runtime/v1alpha3/api.proto

```diff
@@ -1107,6 +1116,35 @@ message StatusResponse {
     map<string, string> info = 2;
 }
 
+// LinuxIDMapping represents a single user namespace mapping in Linux.
+message LinuxIDMapping {
+   // container_id is the starting id for the mapping inside the container.
+   uint32 container_id = 1;
+   // host_id is the starting id for the mapping on the host.
+   uint32 host_id = 2;
+   // size is the length of the mapping.
+   uint32 size = 3;
+}
+
+message LinuxUserNamespaceConfig {
+   // is_enabled, if true indicates that user-namespaces are supported and enabled in the container runtime
+   bool is_enabled = 1;
```

Comment:
I don't think this field is required. An empty list of mappings also means user-namespace is not enabled